### PR TITLE
add support for QUIC protocol

### PR DESCRIPTION
**Pull request by nikooo777**
Beamer and I worked very hard to figure out the issue and to find a solution! Thank you Grin for the suggestion ;)

### CODE REVIEW

**Comment by andybeletsky** [comment]
[Fixed atime](https://github.com/lbryio/reflector.go/pull/38/files#diff-40ac8af7685d24ceb8ef417317dbf053) to work on non-linux platforms.

**Comment by kauffj** [comment]
@lyoshenka booooo 30 day old code in production that you keep making me look at in weekly meetings

File: main.go [hunk 1]
```diff
@@ -5,9 +5,15 @@ import (
 	"time"
 
 	"github.com/lbryio/reflector.go/cmd"
+
+	"github.com/google/gops/agent"
+	log "github.com/sirupsen/logrus"
 )
 
 func main() {
+	if err := agent.Listen(agent.Options{}); err != nil {
```
Comment (lyoshenka) [inline main.go:14]: should this always be on, or just for testing/debugging?

File: peer/quic/client.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+}
+
+// Connect connects to a specific clients and errors if it cannot be contacted.
+func (c *Client) Connect(address string) error {
+	var err error
+	if c.Timeout == 0 {
+		c.Timeout = 5 * time.Second
+	}
+	tlsConf := &tls.Config{
+		InsecureSkipVerify: true,
```
Comment (lyoshenka) [inline peer/quic/client.go:40]: sketchy

**Comment by lyoshenka** [comment]
@kauffj if you want it reviewed then you shoulda put a review request on it

**Pull request closed without merging**